operate/importer/src/main/java/io/camunda/operate/zeebeimport/opensearch/OpensearchRecordsReader.java

```diff
@@ -575,16 +575,30 @@ private boolean isPartitionCompletedImporting(final int partitionId) {
             .query(term(ImportPositionIndex.PARTITION_ID, partitionId))
             .source(sourceInclude(ImportPositionIndex.FIELD_INDEX_NAME))
             .size(50);
-    final var response =
-        zeebeRichOpenSearchClient.doc().search(searchRequestBuilder, ImportPositionEntity.class);
+    try {
+      final var response =
+          zeebeRichOpenSearchClient.doc().search(searchRequestBuilder, ImportPositionEntity.class);
 
-    final var indexNames =
-        response.hits().hits().stream()
-            .map(hit -> hit.source().getIndexName())
-            .filter(Objects::nonNull)
-            .toList();
+      final var indexNames =
+          response.hits().hits().stream()
+              .map(hit -> hit.source().getIndexName())
+              .filter(Objects::nonNull)
+              .toList();
 
-    return indexNames.stream().anyMatch(indexName -> indexName.contains("8.8"));
+      return indexNames.stream().anyMatch(indexName -> indexName.contains("8.8"));
+    } catch (final OpenSearchException e) {
+      LOGGER.warn(
+          "OpenSearch error when checking if partition {} completed importing. Will retry later: {}",
```

Comment:
Omit the part about retrying later, this is logic handled by the calling method and is not a good fit in this catch block